app/jobs/prepaid_credit_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class PrepaidCreditJob < ApplicationJob
```

Comment:
can you namespace it? `Wallets::PrepaidCreditJob` for example?

Reply:
as discussed offline, namespaced it with `Invoices`